### ISSUE TITLE
Problem: multiple podcasts from same domain cannot be differentiated

### ISSUE THREAD

**Issue by frecuencialibre** [LibreTime/libretime#631]
**Problem:** 
the current [album override method](https://github.com/LibreTime/libretime/issues/89) for querying downloaded podcasts with a smartblock uses the domain from which they were downloaded to set the album. 
at our station we need several podcasts uploaded to archive.org. eg: 
https://archive.org/services/collection-rss.php?query=uploader:noticiasdeabajoml@gmail.com
https://archive.org/services/collection-rss.php?query=uploader:latinoamericarompeelcerco@gmail.com
both are giving the same album metadata, and so are impossible to program separately. 

**Proposed solution:** 
from a user's perspective, it would be great to be able to manually specify a text string that would be set in the downloaded podcasts' album metadata. currently a modal dialogue asks you for a url, which then cannot be edited. I'd of course defer to you @Robbt regarding whether any of this is feasible, but think a minimal solution would be to add a "Podcast name" field to the modal. Better yet would be to make the interface more closely match that of playlists and smartblocks: that instead of a modal, it look something like the screenshot below, and enable editing after initial creation. This however is all above and beyond the minimum we'd need as a station to get past this blocker.  thoughts? thanks!

![screenshot from 2018-12-03 17-04-53](https://user-images.githubusercontent.com/36803137/49407302-ab02cd80-f71d-11e8-9825-b0a37bf425d8.png)

**Comment by hairmare** [comment]
Should this stem from some field in the actual RSS/Atom feed? It looks like the archive feeds are a special case here with very generic info in those fields.

**Comment by Robbt** [comment]
Yeah I suspect this is an issue with archive.org but I also think allowing people to override the name of the podcast makes sense. There are lost of podcasts where the metadata isn't great for one reason or another and more accurate information could be manually entered.

The issue I see with this and the autogenerated smartblocks is we would need to generate a new smartblock and playlist everytime there is a name change or the user would need to do this. It probably doesn't make sense to have a button that you would only want to push once. It also would mean that the old smartblock with the old name would still be around. I don't think we'd want to auto delete it. It probably wouldn't be that hard to check the preferences to see if autogenerate smartblocks was setup and then regenerate it if they change the name of a smartblock. 

It also wouldn't update any tracks that were already downloaded with the old name and I'm pretty sure we wouldn't want to do that. We'd also have to look at the celery task that does the downloading and modify where it gets the name since it is the place that renames the album metadata.

I know that @ned-kelly was talking about putting a PR in with his ffmpeg workaround for m4a files since evidently liquidsoap 1.1.1 and 1.3.x fail with m4a files based upon his testing. It might make sense to work on all of this at the same time.
